src/arch/riscv/rvdbg013.c

```diff
@@ -1490,11 +1495,11 @@ static bool riscv_check_watch(target *t, target_addr *watch)
 		/* 'hit' bit unimplemented, instruction decoding fallback */
 		DEBUG_TARGET("hit bit unimplemented\n");
 		if (decode_load_store_inst(t, dpc, watch)) {
-	for (bw = t->bw_list; bw; bw = bw->next) {
-		if ((bw->type == TARGET_WATCH_WRITE) ||
-		    (bw->type == TARGET_WATCH_READ) ||
-		    (bw->type == TARGET_WATCH_ACCESS)) {
-			if (bw->addr == *watch) {
+			for (bw = t->bw_list; bw; bw = bw->next) {
```

Comment:
Nitpicking: Possible whitespace change

Reply:
I have tabs here, indentation sounds good on my side. Is it line 1500 that you are referring to?

Reply:
Maybe it is the same as your comment at line 1597?
(bw->type == TARGET_WATCH_READ) is shifted with 4 little spaces to align it with (bw->type == TARGET_WATCH_WRITE) in every editor configuration.

Reply:
Ah., I see, indentation was not right with the first try an now you corrected it. Probably your tab width is > 4 space  and BMP uses 4 in most places, beside lapses slipped in.